contracts/RelayHub.sol

```diff
@@ -23,7 +26,7 @@ contract RelayHub is IRelayHub {
     * Assume that relay has non-zero balance (costs 15'000 more otherwise).
     */
     uint256 constant public gasReserve = 100000; // how much reserve we actually need, to complete the post-call part of relayCall().
-    uint256 constant public gasOverhead = 47241;
```

Comment:
I'm not sure, since the only change is the addition of this array (which has length 0 in the test that measures this overhead). We're in the process of measuring the gas consumed by each block of code in `relayCall`, we can do a more in-depth analysis of gas expenditure once those measurements are finalized.